docs/video/ffmpeg/upscaling.md

```diff
@@ -50,12 +50,13 @@ Here are two videos, one was left as is, the other one stretched to 4K, watch it
     
 === "1080p stretched to 4K"
 
-    <iframe width="688" height="387" src="https://www.youtube.com/embed/-FE9s_acdYw?color=white" frameborder=0 allowfullscreen></iframe>
+    <iframe width="688" height="387" src="https://www.youtube.com/embed/-FE9s_acdYw" frameborder=0 allowfullscreen></iframe>
 
     
 ## **Why not just set the project scale to 4K before rendering?** {#why-not-nle}
 
-VEGAS Pro (and probably other NLEs) uses the bicubic scaling filter which makes your video a bit blurry, using FFmpeg lets you make sure you're using the right filter
+VEGAS Pro (and probably any other NLE) uses the bicubic scaling filter which makes your video a bit blurry, using FFmpeg lets you make sure you're using the right filter
```

Comment:
where do you source that from, that makes bicubic much more prominent

Reply:
i mainly did it to stay consistent with the plurals (vegas pro and NLE (singular)), as previously it was vegas pro and NLES (plural). i agree that "any other" could have probably been phrased better (e.g. "a lot of other)"